op-acceptance-tests/tests/base/chain/init_test.go

```diff
@@ -0,0 +1,15 @@
+package chain
+
+import (
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/compat"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+)
+
+func TestMain(m *testing.M) {
+	presets.DoMain(m,
+		presets.WithCompatibleTypes(compat.SysGo),
```

Comment:
Lets add a TODO here with a Github issue to fix this incompatibility.
Why doesn't it work on Kurtosis and Persistent?